classes/Country.php

```diff
@@ -176,7 +176,7 @@ public static function getCountriesByIdShop($idShop, $idLang)
      * Get a country ID with its iso code.
      *
      * @param string $isoCode Country iso code
-     * @param bool $active return only active coutries
+     * @param bool $active return only active countries
```

Comment:
Hi @Arnaud104, thanks for your contribution !
Can you please fix here to https://github.com/Arnaud104/PrestaShop/blob/patch-1/classes/Country.php#L133